camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/CamusJobTestWithMock.java

```diff
@@ -151,7 +154,15 @@ public void runJob1() throws Exception {
   }
 
   private void setupJob1() {
```

Comment:
Broke the original setupJob1() into multiple small methods, each creating one mock, so that they can be reused.